src/app/sol-warnings.js

```diff
@@ -0,0 +1,34 @@
+// source: just a big string of the current tab's source code.
+//
+// compiledResult: an object with different keys based on success/failure of compilation.
+//   success keys: 'contracts' and 'sources' (contains AST)
+//   error key: 'error' (contains compile error string)
+
+function find (source, compiledResult) {
+  var messages = [];
+
+  if (source) {
+    if (source.match(/\.\s*send\s*\(/)) {
+      messages.push('Caution: You are using the send method.\nPlease read <a target="_blank" href="http://hackingdistributed.com/2016/06/16/scanning-live-ethereum-contracts-for-bugs/">this</a> before deploying your code.');
```

Comment:
I think it would be better to link to the documentation, also the same warning could be issued for any function call.

Also what about adding a general warning "Interacting with unknown contracts (including sending ether) can cause arbitrary side-effects including recursive calls into the current contract" before the call returns."?